eng/doc/CrossPlatformCryptography.md

```diff
@@ -79,83 +84,88 @@ This section includes the following packages:
 
 This section includes the following packages:
 
-* [crypto/aes](https://pkg.go.dev/crypto/aes)
-* [crypto/cipher](https://pkg.go.dev/crypto/cipher)
-* [crypto/des](https://pkg.go.dev/crypto/des)
-* [crypto/rc4](https://pkg.go.dev/crypto/rc4)
-
-| Cipher + Mode | Windows  | Linux            |
-|---------------|----------|------------------|
-| AES-ECB       | ✔️       | ✔️              |
-| AES-CBC       | ✔️       | ✔️              |
-| AES-CTR       | ❌       | ✔️              |
-| AES-CFB       | ❌       | ❌              |
-| AES-OFB       | ❌       | ❌              |
-| AES-GCM       | ✔️       | ✔️              |
-| DES-CBC       | ✔️       | ⚠️<sup>1</sup>  |
-| DES-ECB       | ✔️       | ⚠️<sup>1</sup>  |
-| 3DES-ECB      | ✔️       | ✔️              |
-| 3DES-CBC      | ✔️       | ✔️              |
-| RC4           | ✔️       | ⚠️<sup>1</sup>  |
+- [crypto/aes](https://pkg.go.dev/crypto/aes)
+- [crypto/cipher](https://pkg.go.dev/crypto/cipher)
+- [crypto/des](https://pkg.go.dev/crypto/des)
+- [crypto/rc4](https://pkg.go.dev/crypto/rc4)
+
+| Cipher + Mode | Windows | Linux          | macOS |
+| ------------- | ------- | -------------- | ----- |
+| AES-ECB       | ✔️      | ✔️             | ✔️    |
+| AES-CBC       | ✔️      | ✔️             | ✔️    |
+| AES-CTR       | ❌      | ✔️             | ❌    |
+| AES-CFB       | ❌      | ❌             | ❌    |
+| AES-OFB       | ❌      | ❌             | ❌    |
+| AES-GCM       | ✔️      | ✔️             | ✔️    |
+| DES-CBC       | ✔️      | ⚠️<sup>1</sup> | ✔️    |
+| DES-ECB       | ✔️      | ⚠️<sup>1</sup> | ✔️    |
+| 3DES-ECB      | ✔️      | ✔️             | ✔️    |
+| 3DES-CBC      | ✔️      | ✔️             | ✔️    |
+| RC4           | ✔️      | ⚠️<sup>1</sup> | ✔️    |
 
 <sup>1</sup>When using OpenSSL 3, requires the legacy provider to be enabled.
 
 ### AES-GCM keys, nonces, and tags
 
-* Key Sizes
+- Key Sizes
 
   AES-GCM works with 128, 192, and 256-bit keys.
 
-* Nonce Sizes
+- Nonce Sizes
 
   AES-GCM works with 12-byte nonces.
 
-* Tag Sizes
-  
+- Tag Sizes
+
   AES-GCM works with 16-byte tags.
 
 ## Asymmetric encryption
 
 This section includes the following subsections:
 
-* [RSA](#rsa)
-* [ECDSA](#ecdsa)
-* [ECDH](#ecdh)
-* [Ed25519](#ed25519)
-* [DSA](#dsa)
+- [RSA](#rsa)
+- [ECDSA](#ecdsa)
+- [ECDH](#ecdh)
+- [Ed25519](#ed25519)
+- [DSA](#dsa)
 
 ### RSA
 
+> [!WARNING]  
+> System Libraries might not fully implement all RSA features. Please see the subsections below for more information.
```

Comment:
Why a warning for RSA in particular? Many of the tables have feature gaps explained below them.

Also:

```suggestion
> System libraries might not fully implement all RSA features. Please see the subsections below for more information.
```